osbs/build/build_request.py

```diff
@@ -86,6 +88,7 @@ def set_params(self, **kwargs):
                                    (private, authoritative-source-only, restricted, public)
         :param use_auth: bool, use auth from atomic-reactor?
         :param low_priority_node_selector: dict, a nodeselector for builds with lower priority
+        :param platform_node_selector: dict, a nodeselector for builds with lower priority
```

Comment:
Should be "for a specific platform".

Do we set `self.platform_node_selector` to the value that is passed in?